### PR TITLE
Render deployment check results into the app listing.

### PR DESCRIPTION
**Pull request by aquamatthias**
Attention: this PR is based on PR #3617. Once this one is merged I can rebase on the feature branch.

### CODE REVIEW

File: docs/docs/rest-api/public/api/v2/groups.raml [hunk 1]
```diff
@@ -24,6 +24,8 @@ get:
 
         - <code>group.apps.deployments</code> embed all deployment identifier, if the related app currently is in deployment.
 
+        - <code>group.apps.readiness</code> embed all readiness results, if the related app has readines checks defined.
```
Comment (gkleiman) [inline docs/docs/rest-api/public/api/v2/groups.raml]: I'd replace this with:

`embed all app readiness results`

File: docs/docs/rest-api/public/api/v2/apps.raml [hunk 2]
```diff
@@ -49,6 +49,8 @@ get:
 
         - <code>apps.deployments</code> embed all deployment identifier, if the related app currently is in deployment.
 
+        - <code>apps.readiness</code> embed all readiness results, if the related app has readines checks defined.
```
Comment (gkleiman) [inline docs/docs/rest-api/public/api/v2/apps.raml]: If there are no readiness checks, then .

The clarification led me to thing that if the app definition doesn't define any readiness checks, then nothing will be added to the result. But actually Marathon will add an empty array.

So I'd just remove the clarification and leave this as `embed all readiness check results`.

**Comment by gkleiman** [comment]
@aquamatthias LGTM.

Just a little nit about the documentation. Please rebase and I'll gladly merge.

**Comment by aquamatthias** [comment]
wrong branch. close.

**Pull request closed without merging**